blueocean-core-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jenkins-cd/blueocean-core-js",
-  "version": "0.0.45",
+  "version": "0.0.46-unpublished",
```

Comment:
that is new. Until now we used the "maven" way. Things in master are "snapshots" if that has changed then we need to make this public knowledge.